client/components/domains/reskin-side-explainer/index.jsx

```diff
@@ -101,6 +101,12 @@ class ReskinSideExplainer extends Component {
 				ctaText = translate( 'Use a domain I own' );
 				break;
 
+			case 'site-migration':
```

Comment:
Do we have translations for these strings already or do we need to wrap them for English locales before shipping?

Reply:
I wrapped them for English locales until the translations are complete.